src/component/Label.tsx

```diff
@@ -483,47 +490,53 @@ const parseViewBox = (props: any): ViewBox => {
   return {};
 };
 
-const parseLabel = (label: unknown, viewBox: ViewBox) => {
+const parseLabel = (label: unknown, viewBox: ViewBox, labelRef?: React.RefObject<Element>) => {
   if (!label) {
     return null;
   }
 
+  const commonProps = { viewBox, labelRef };
+
   if (label === true) {
-    return <Label key="label-implicit" viewBox={viewBox} />;
+    return <Label key="label-implicit" {...commonProps} />;
   }
 
   if (isNumOrStr(label)) {
-    return <Label key="label-implicit" viewBox={viewBox} value={label} />;
+    return <Label key="label-implicit" value={label} {...commonProps} />;
   }
 
   if (isValidElement(label)) {
     if (label.type === Label) {
-      return cloneElement<LabelProps>(label, { key: 'label-implicit', viewBox });
+      return cloneElement<LabelProps>(label, { key: 'label-implicit', ...commonProps });
     }
 
-    return <Label key="label-implicit" content={label} viewBox={viewBox} />;
+    return <Label key="label-implicit" content={label} {...commonProps} />;
```

Comment:
when `label` is an HTML or SVG element `label` gets passed to `content` and then `content` is cloned with `props`. `props` includes `labelRef` which isn't a valid prop in the DOM